common/src/main/java/com/wynntils/models/items/encoding/data/NameData.java

```diff
@@ -6,9 +6,22 @@
 
 import com.wynntils.models.items.encoding.type.ItemData;
 import com.wynntils.models.items.properties.IdentifiableItemProperty;
+import java.util.regex.Pattern;
 
 public record NameData(String name) implements ItemData {
+    private static final int MAX_NAME_LENGTH = 48;
+    private static final Pattern SANITIZE_PATTERN = Pattern.compile("[^a-zA-Z0-9'\\s]");
```

Comment:
Are you sure `-` is not allowed? If not, names like `Uruk-Hai Blade` would not be allowed.

Reply:
![image](https://github.com/user-attachments/assets/9b539490-37f8-436f-b35b-7e43dfa71aa8)

Numbers are allowed even though it says otherwise in the message

Reply:
Ok!